### PR TITLE
fix(standalone): shutdown server on initialization errors

### PR DESCRIPTION
**Pull request by aandis**
Fixes the following -

1. Server not shutting down on initialization errors.
2. NPE during shutdown due to http server not being initialised.

**Pull Request checklist**

- [x] The commit(s) message(s) follows the contribution [guidelines](CONTRIBUTING.md) ?
- [x] Tests for the changes have been added (for bug fixes / features) ?
- [x] Docs have been added / updated (for bug fixes / features) ?

### CODE REVIEW

File: standalone/src/main/scala/filodb.standalone/FiloServer.scala [hunk 1]
```diff
@@ -90,10 +91,6 @@ class FiloServer(watcher: Option[ActorRef]) extends FilodbClusterNode {
 
 object FiloServer extends StrictLogging {
   def main(args: Array[String]): Unit = {
-    try {
-      new FiloServer().start()
-    } catch { case e: Exception =>
-      logger.error("Could not start FiloDB server", e)
-    }
+    new FiloServer().start()
```
Comment (vishramachandran) [inline standalone/src/main/scala/filodb.standalone/FiloServer.scala:94]: We need a force shutdown here. There is a possibility of daemon threads holding up the process.

Comment (aandis) [reply standalone/src/main/scala/filodb.standalone/FiloServer.scala:94]: The force shutdown is already present at https://github.com/filodb/FiloDB/pull/1124/files#diff-c50749986da7e22c6df371808ceb7bfed314508b866b9a0db2d9bcd4727165e1R72-R75

Do we still need to add it here?

Comment (vishramachandran) [reply standalone/src/main/scala/filodb.standalone/FiloServer.scala:94]: Ah got it. Do you mind moving the try-catch here to the uppermost call - that way any future additions here will also benefit from it. Thanks.

**Review by vishramachandran — APPROVED** [review]
Approved with suggestion

**Pull request merged**